great-idea/css/index.css

```diff
@@ -63,4 +63,196 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+/* ----Main Layout Section---- */
```

Comment:
This separation of concerns and commenting that you are starting to do here is great - this is the kind of organizational  best practices that you want to carry through everything that you do from here on out!